hibernate-reactive-core/src/main/java/org/hibernate/reactive/vertx/impl/ProvidedVertxInstance.java

```diff
@@ -13,7 +13,7 @@
 
 /**
  * An implementation of {@link VertxInstance} which allows the client
- * to provide an instance of {@link Vertx} whose lifecyle is managed
+ * to provide an instance of {@link Vertx} whose lifecycle is managed
```

Comment:
It's not part of your changes, but a few lines after this it states:

> Hibernate will destroy the {@code Vertx} instance on shutdown.

This is not true though, isn't it?
a `ProvidedVertxInstance` won't do anything to the instance when the factory is closed

Reply:
I think it's copied from `DefaultVertxInstance`, since you are alredy updating this, could fix this too, please?

Reply:
Yah, looks wrong.